package.json

```diff
@@ -59,6 +59,7 @@
     "react-native-svg": "^9.8.4",
     "react-native-touch-id": "^4.4.1",
     "react-native-webview": "git+https://github.com/status-im/react-native-webview.git#v10.2.3_5",
+    "tdigest": "^0.1.1",
```

Comment:
could you please elaborate why we need this new dep ?

Reply:
This is used for statisitcs inside profiller

Reply:
yeah, i saw that, i mean could we avoid using it? we should try to avoid adding new dependencies because of security reasons at first

Reply:
Without it, it will be not possible to see live updating of render statistics. The only way will be to accumulate all data and calculate it at the end. But to calculate we will again want to use a lib cause for math. This library should not go into release as the namespace is not required and compiler will remove it.